docs/contributing/development/continuous_integration.rst

```diff
@@ -292,8 +44,7 @@ Testing pipeline
 ================
 
 The `testing pipeline`_ (CI) consists basically in the same job running twice
```

Comment:
6 jobs in parallel(continuum and non continuum tests each times 3 different OS platforms)

Reply:
osx arm